src/metric_model_reporter.h

```diff
@@ -128,10 +125,9 @@ class MetricModelReporter {
   prometheus::Counter* metric_inf_compute_infer_duration_us_;
   prometheus::Counter* metric_inf_compute_output_duration_us_;
   prometheus::Counter* metric_cache_hit_count_;
-  prometheus::Counter* metric_cache_hit_lookup_duration_us_;
+  prometheus::Counter* metric_cache_hit_duration_us_;
   prometheus::Counter* metric_cache_miss_count_;
-  prometheus::Counter* metric_cache_miss_lookup_duration_us_;
-  prometheus::Counter* metric_cache_miss_insertion_duration_us_;
+  prometheus::Counter* metric_cache_miss_duration_us_;
```

Comment:
Are we removing one of the metrics from the cache API? Looks like we no longer have `metric_cache_miss_insertion_duration_us_`.

Reply:
Under the hood, these metrics were combined in the `infer_stats` endpoint (what's reported to perf_analyzer)

cache_miss insert+lookup are being combined into one for consistency `cache_hit` metric, consistency with what infer_stats reports, and less clutter in metrics output.